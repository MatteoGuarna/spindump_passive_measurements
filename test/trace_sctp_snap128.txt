A simple SCTP test where traces are captured with snaplen equal to 128.
