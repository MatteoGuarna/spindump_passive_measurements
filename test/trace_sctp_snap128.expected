SCTP 127.0.0.1 <-> 127.0.0.1 502339778:0 (60782:62324) at 1581069557459390 new starting packets 1 0 bytes 84 0 bandwidth 84 0
SCTP 127.0.0.1 <-> 127.0.0.1 502339778:1079218523 (60782:62324) at 1581069557459633 measurement up right 12 packets 3 2 bytes 444 376 bandwidth 444 376
SCTP 127.0.0.1 <-> 127.0.0.1 502339778:1079218523 (60782:62324) at 1581069557459758 measurement up left 82 packets 3 4 bytes 444 488 bandwidth 444 488
